### PR TITLE
docs: rename stories names to be in pascal case

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/5507683216

### CODE REVIEW

File: src/components/Combobox/__stories__/combobox.stories.mdx [hunk 1]
```diff
@@ -129,7 +129,7 @@ Default combo box can be used without dialog or as part of the layout.
 
 ### Comobox inside a dialog
 
-Use this for combo box that triggered by button.
+Use this for Combobox that triggered by button.
 
 <Canvas>
   <Story name="Comobox inside a dialog">
```
Comment (talkor) [inline src/components/Combobox/__stories__/combobox.stories.mdx:135]: Typo

**Review by talkor — APPROVED** [review]

File: src/components/Combobox/__stories__/combobox.stories.mdx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Combobox/__stories__/combobox.stories.mdx:135]: it was intentional)

Comment (talkor) [reply src/components/Combobox/__stories__/combobox.stories.mdx:135]: No no, `Comobox` lol

Comment (SergeyRoyt) [reply src/components/Combobox/__stories__/combobox.stories.mdx:135]: Aahh, now I see it :D
Will fix, thanks!

**Pull request merged**